tests/SIL.LCModel.Core.Tests/SIL.LCModel.Core.Tests.csproj

```diff
@@ -6,6 +6,7 @@
     <Description>The liblcm library is the core FieldWorks model for linguistic analyses of languages. Tools in this library provide the ability to store and interact with language and culture data, including anthropological, text corpus, and linguistics data.
 This package provides unit tests for SIL.LCModel.Core.</Description>
     <IsPackable>true</IsPackable>
+	  <Nullable>disable</Nullable>
```

Comment:
indenting is weird

Reply:
Ah, I must have some Rider settings weird